internal/backends/winit/lib.rs

```diff
@@ -24,6 +24,8 @@ mod renderer {
 
     use i_slint_core::window::PlatformWindow;
 
+    mod boxshadowcache;
```

Comment:
This is not a renderer.  Should this module be in the root?

Reply:
Like the `SoftwareRenderer` in the `platform`? ;-) I was thinking that this is only used among renderers. But if you prefer I can move it into the root.